core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -525,6 +525,9 @@
   TSERV_SLOW_FLUSH_MILLIS("tserver.slow.flush.time", "100ms", PropertyType.TIMEDURATION,
       "If a flush to the write-ahead log takes longer than this period of time,"
           + " debugging information will written, and may result in a log rollover."),
+  TSERV_SLOW_FILEPERMIT_MILLIS("tserver.slow.filepermit.time", "100ms", PropertyType.TIMEDURATION,
```

Comment:
I think it would be good to add this to the set HOT_PATH_PROPERTIES as it will be checked on a per scan basis.

Reply:
okay